CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Creating Pre/Releases For A Model Deployment Repository
+
+This model deployment repository makes use of the [spack](https://spack.readthedocs.io/en/latest/) build and deployment functionality enabled by the [access-nri/build-cd repository](https://github.com/ACCESS-NRI/build-cd).
+
+This allows automated Release and Prerelease `spack` builds of climate models, informed by this repositories `spack.yaml` file and `config` directory.
+
+In a nutshell, the commits on Pull Requests to this model deployment repository generate isolated Prerelease builds of a climate model on HPCs. When these are merged, an official Release build is created.
+
+FIXME: Add in the image of the process
+
+## The `spack.yaml` File
+
+The `spack.yaml` file at the root of the repository is a file understood by `spack`. It constrains the versions and features (variants) of dependencies required to build a given climate model. It is a set of abstract constraints that is _concretized_ into a single set of dependencies at build time, creating a `spack.lock` file.
+
+It is similar to `npm`s `package.json`/`package.lock` files.
+
+More information on the `spack.yaml` file itself can be found on [ACCESS-NRI's devdocs](https://github.com/ACCESS-NRI/dev-docs/wiki/Spack#the-spackyaml-file-spec-file).
+
+## The `config` Directory
+
+The `config` directory at the root of this repository contains a single `versions.json` file, which allows customisation of both the version of [`access-nri/spack`](https://github.com/ACCESS-NRI/spack) that the model will be deployed on, as well as the version of [`access-nri/spack-packages`](https://github.com/ACCESS-NRI/spack-packages) that will source the ACCESS-NRI spack packages recipes.
+
+## The PR Process
```

Comment:
What we require is a document that demonstrates what the user needs to do if they wish to modify a package's source code. e.g. The user modifies MOM5 source code, creates a branch, pushes it to ACCESS-NRI's MOM5 repo. Then they have to modify `spack.yaml` and change the MOM5 version to `git.<branch>`, etc.

Reply:
Maybe it would be worth having a chat about this in a call.

Reply:
I'd suggest changing the title to something like "Updating the Model" and saying underneath: "Updating the model requires a pull request (PR) process".

Reply:
```suggestion
## Changing the Model via Pull Request (PR)
```